packages/midt/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mapspeople/midt.git"
+    "url": "git://github.com/MapsPeople/web-ui.git"
```

Comment:
Alright, I wanted to educate myself a bit, and see if there was a smarter way to do this 😀

So, I checked out the React Native repo for inspiration (I just wanted a popular one), and found this:

[`"repository": "github:facebook/react-native",`](https://github.com/facebook/react-native/blob/0c9c57a9f73294414d92428c5d2472dc1e1e5e96/package.json#L7)

The shorthand for `github:mapspeople/web-ui` would be nice, but there is an even better way.

Reading the docs for `package.json` on npm, they have this example:

```
{
  "repository": {
    "type": "git",
    "url": "https://github.com/facebook/react.git",
    "directory": "packages/react-dom"
  }
}
```

Which is how you link to a package inside a monorepo 🎉 

So, my suggestion is to have these lines like so (but I can't suggest it because multi-line suggestions on deleted lines is not possible here 🤷):

```
  "repository": {
    "type": "git",
    "url": "https://github.com/MapsPeople/web-ui.git",
    "directory": "packages/midt"
  }
```

Reply:
That's pretty nice! 🥳 I updated the package.json with the suggested changes.